### PR TITLE
Update ar.js

### PR DESCRIPTION
**Pull request by xahmd**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: ui/lang/ar.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default {
     update: 'تحديث',
     create: 'إنشاء',
     search: 'بحث',
-    filter: 'ترشيح',
+    filter: 'تنقية',
```
Comment (hawkeye64) [inline ui/lang/ar.js:17]: Google is really bad for translating these. But just for reference, the removed one comes up as "nomination". The added one comes up as "Purification" (maybe somewhat of a filter). 
@ahmedbinmoh I am wondering if this one would be suitable "منقي" (purifier)

Comment (xahmd) [reply ui/lang/ar.js:17]: the old one its means "nomination" , but the new it means "filter" or "Selection" 
google translate its not 100% correct, so we cant walk with .

**Comment by rstoenescu** [comment]
Thanks for contributing!
This will go into v2.0.1 and v1.15.22.

**Comment by ejez** [comment]
كلمة التنقية أو التصفية مستمدة من النقاء أو الصفاء، وهو ما يترجم إلى
 'being clean'
 بالانجليزية، وهذا المعنى غير مناسب في هذا السياق.

كلمة ترشيح تعني تمرير شيء ما عبر مرشح، وما يرشح يحتفظ به، وهذه الكلمة أكثر استعمالا في مجال التقنية للدلالة على 
'filtering'
انظر كمثال
https://www.almaany.com/ar/dict/ar-en/filtering/

The words 'تنقية' أو 'تصفية' relate more to 'cleaning' rather than 'filtering'.

The meaning of 'ترشيح' is exactly what we need, it means passing something through a filter ('مرشح').

Of course in any language the same word can have multiple meanings, 'nomination' is another use for 'ترشيح' but it is not the intended meaning in our context.

In the tech domain the word 'ترشيح' is widely more used in the context of filtering: https://www.almaany.com/ar/dict/ar-en/filtering/

**Comment by pdanpdan** [comment]
@ejez can you please make a PR for that, we don't know the language. Thank you

**Comment by ejez** [comment]
@ahmedbinmoh , @alonemazin what are your thoughts about this.

**Comment by ejez** [comment]
note also that wikipedia uses 'ترشيح'
https://en.m.wikipedia.org/wiki/Filter

https://ar.m.wikipedia.org/wiki/%D9%85%D8%B1%D8%B4%D8%AD_(%D8%AA%D9%88%D8%B6%D9%8A%D8%AD)

**Pull request merged**